### PR TITLE
Clean up script documentation

### PR DESCRIPTION
**Pull request by GriffinRichards**

### CODE REVIEW

File: data/specials.inc [hunk 1]
```diff
@@ -13,164 +13,164 @@ gSpecials:: @ 815FD60
...
+	def_special NullFieldSpecial
+	def_special NullFieldSpecial
 	def_special IsEnigmaBerryValid
-	def_special ScrSpecial_GetTrainerBattleMode
-	def_special ScrSpecial_ShowTrainerIntroSpeech
-	def_special ScrSpecial_ShowTrainerNonBattlingSpeech
+	def_special GetTrainerBattleMode
+	def_special ShowTrainerIntroSpeech
+	def_special ShowTrainerCantBattleSpeech
 	def_special ScrSpecial_HasTrainerBeenFought
```
Comment (jiangzhengwenjz) [inline data/specials.inc]: maybe we can remove all `Special_` `ScrSpecial_` prefixes?

Comment (GriffinRichards) [reply data/specials.inc]: I left a few in. Some can probably be removed, others like this one currently have duplicate names (`HasTrainerBeenFought` already exists)

File: data/text/poke_dude.inc [hunk 2]
```diff
@@ -1,154 +1,129 @@
-gUnknown_81C5F69::
+PokeDude_Text_SpeedierBattlerGoesFirst::
```
Comment (jiangzhengwenjz) [inline data/text/poke_dude.inc]: `poke_dude` and `PokeDude` are inconsistent with `pokedude` and `Pokedude` used in other locations

File: src/help_system_812B1E0.c [hunk 3]
```diff
@@ -35,415 +35,415 @@ static void ResetHelpSystemCursor(struct HelpSystemListMenu * a0);
 static void PrintHelpSystemTopicMouseoverDescription(struct HelpSystemListMenu * a0, struct ListMenuItem * a1);
 
 static const u8 *const sHelpSystemTopicPtrs[] = {
-    gUnknown_81B2DF8,
```
Comment (PikalaxALT) [inline src/help_system_812B1E0.c]: Would make sense to define enums for these and the other arrays.

File: include/constants/help_system.h [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+#define HELPCONTEXT_PLAYERS_PC_ITEMS 29
+#define HELPCONTEXT_PLAYERS_PC_MAILBOX 30
+#define HELPCONTEXT_PC_MISC 31 // Prof Oaks PC and the HoF PC
+#define HELPCONTEXT_BEDROOM_PC 32
+#define HELPCONTEXT_BEDROOM_PC_ITEMS 33
+#define HELPCONTEXT_BEDROOM_PC_MAILBOX 34
+#define HELPCONTEXT_UNUSED 35
+#define HELPCONTEXT_COUNT  36
+
+#endif //GUARD_CONSTANTS_HELP_SYSTEM_H
```
Comment (PikalaxALT) [inline include/constants/help_system.h]: Missing newline at end of file

File: src/help_system_812B1E0.c [hunk 5]
```diff
@@ -16,811 +16,1678 @@
...
-    gUnknown_81B3440,
-    gUnknown_81B3457,
-    gUnknown_81B346F,
-    gUnknown_81B3481,
-    gUnknown_81B349B,
-    gUnknown_81B34B7,
-    gUnknown_81B34D6,
-    gUnknown_81B34F6,
-    gUnknown_81B3516
+    Help_Text_PlayingForFirstTime,
```
Comment (PikalaxALT) [inline src/help_system_812B1E0.c]: Use the above enums to initialize this and the next array by index

**Pull request merged**